src/js/api.js

```diff
@@ -7,6 +7,12 @@ export class API {
     this.baseUrl = sessionStorage.getItem("apiBaseUrl") || defaultBaseUrl;
   }
 
+  getMyMuni() {
+    const myMuniPagesUrl = "https://muni-portal-backend.openup.org.za/api/wagtail/v2/pages/?child_of=11";
```

Comment:
we shouldn't hardcode IDs - how about doing two requests - one for a page of type MyMuni, get its ID from the response, then a request for children of that page?

Reply:
Ah, that works. I thought you meant to use the URL you provided for now 👍